contributors/design-proposals/optional-configmap.md

```diff
@@ -0,0 +1,111 @@
+# Optional ConfigMaps
+
+## Goal
+
+Allow the ConfigMaps that are used to populate the environment variables of a
+container and files within a Volume to be optional.
+
+## Use Cases
+
+An application developer can define a set of environment variables for a
+container by referencing a ConfigMap with default values. The developer can
+also define an optional ConfigMap that is used to override the values. There
+may be a set of these optional ConfigMaps that provide environment specific
+overrides for a given test cluster or region cluster. If no overrides are
+present, the container will get the default values.
```

Comment:
> the container will get the default values

Not sure what "default values" means here. Is this the "define literally, then define optional configmap overrides" case I asked about below?